test/emitting-events.spec.ts

```diff
@@ -1,7 +1,11 @@
-import {ExportedCellChange, ExportedNamedExpressionChange, HyperFormula} from '../src'
-import {ErrorType} from '../src/Cell'
+import {
```

Comment:
Is this change supposed to be present in this PR? It seems unrelated

Reply:
Yes, it is unrelated, but this is a unit test that I created while working on answer to this discussion: https://github.com/handsontable/hyperformula/discussions/1152

It's beneficial to keep this test in the codebase. Do you think I should have created a separate PR for adding it?

Reply:
Yes, let's split it to 2 separate PRs, just for clarification. The PR clearly states it's just a copyright notice, but new test was added as well.

Reply:
Ok, I removed this unit test from this branch